model/BUILD.bazel

```diff
@@ -89,6 +89,7 @@ proto_library(
 
 java_lite_proto_library(
     name = "profile_java_proto_lite",
+    visibility = ["//visibility:public"],
```

Comment:
Is this required? As much as possible we would like to avoid setting visibility to public

Reply:
@vinitamurthi it was written in the issue to make it public. But I'll try API.

Reply:
@vinitamurthi changed visibility label to oppia_api_visibilty from public